src/tools/compiletest/src/runtest.rs

```diff
@@ -214,10 +215,10 @@ impl<'test> TestCx<'test> {
             self.fatal_proc_rec("compilation failed!", &proc_res);
         }
 
+        // FIXME(#41968): Remove this check after #41968 is merged.
         let expected_errors = errors::load_errors(&self.testpaths.file, self.revision);
-        if !expected_errors.is_empty() {
-            self.check_expected_errors(expected_errors, &proc_res);
-        }
+        assert!(expected_errors.is_empty(),
+                "run-pass tests with expected warnings should be moved to ui-run/");
```

Comment:
@Mark-Simulacrum The `//~` are loaded in `errors::load_errors`, and checked against the output in `check_expected_errors`. So they were verified at this line exactly.